build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildTask.java

```diff
@@ -87,6 +80,7 @@
 
     BuildTask(BuildCoordinator buildCoordinator, 
             BuildConfiguration buildConfiguration, 
+            BuildConfigurationAudited buildConfigurationAudited,
```

Comment:
What about adding a BuildConfigurationVersion field instead of audited entity to avoid duplication?